lib/game_of_life_web/live/universe_live.ex

```diff
@@ -0,0 +1,62 @@
+defmodule GameOfLifeWeb.UniverseLive do
+  use Phoenix.LiveView
+  use Phoenix.HTML, only: raw/1
+  alias GameOfLife.Universe
+
+  def render(assigns) do
+    ~L"""
+    <div id="universe-page">
+      <h1>It's time for... the Game... of... LIFE!</h1>
+
+      <div class="universe">
+        <%= for row <- @cells do %>
+          <div class="cell-row">
+            <%= for cell <- row do %>
+              <%= raw cell %>
+            <% end %>
+          </div>
+        <% end %>
+      </div>
+    </div>
+    """
+  end
+
+  def mount(_session, socket) do
+    if connected?(socket), do: :timer.send_interval(500, self(), :tick)
+
+    socket = assign(socket,
+      universe: rand_bytes(),
+      dimensions: {8, 8},
+    )
+
+    Universe.Supervisor.start_child(
+      socket.assigns.universe,
+      socket.assigns.dimensions
+    )
+
+    {:ok, put_cells(socket, &Universe.info/1)}
```

Comment:
Need the cells themselves to start, hence the `info` here, but each tick was fixed to use cells directly returned